### PR TITLE
SCRM-232 Filtering - SuiteR Theme

### PR DESCRIPTION
**Pull request by gymad**
(ref. SCRM-232 Filtering - SuiteR Theme)
## Description

(1. Styling for an empty module i.e. no records yet could be better.)
2. BULK ACTION should be in capitals?  The Font also looks inconsistent.
3. More padding to be made between the tabs on the Pop up.  Also we have a duplicate link to the Quick Filter tab at the bottom.
(4.  User Profile list view shows Mass Update rather than Bulk Actions)
5.   There is an extra faint line underneath the tabs.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [ ] My code follows the code style of this project found [here](https://suitecrm.com/wiki/index.php/Coding_Standards).
- [ ] My change requires a change to the documentation.
- [ ] I have read the [**How to Contribute**](https://suitecrm.com/wiki/index.php/Contributing_to_SuiteCRM) guidelines.

### CODE REVIEW

File: themes/SuiteP/css/listview.scss [hunk 1]
```diff
@@ -1612,15 +1612,11 @@ div#search_form > .advanced input.button:hover {
 
 label.hidden-mobile.selected-actions-label {
   margin-bottom: initial;
-  font-size: 10.2px;
```
Comment (samus-aran) [inline themes/SuiteP/css/listview.scss]: Do we need changes in a different theme if the issue is on SuiteR?

File: themes/Suite7/css/style.css [hunk 2]
```diff
@@ -6499,9 +6499,7 @@ select,
 label.hidden-mobile.selected-actions-label,
 label.hidden-desktop.selected-actions-label {
   margin-bottom: initial;
-  font-size: 10.2px;
```
Comment (samus-aran) [inline themes/Suite7/css/style.css]: Do we need changes in a different theme if the issue is on SuiteR?

**Comment by samus-aran** [comment]
Issue 1:

Still see Mass Update on User's Profile rather than Bulk Action.

suiter3138


Issue 2:

Keep the same font as the Select text beside it: 1em.  The reason why the font looks inconsistent was the letter spacing and the font weight.  If you remove these I think they look better.

suiter3138_1

**Comment by Mac-Rae** [comment]
Please resolve the conflicts on this pull request bringing it up to date

**Comment by Dillon-Brown** [comment]
@Mac-Rae Actually this can just be closed as it's SuiteR specific

**Pull request closed without merging**